xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/displayer_page.vm

```diff
@@ -55,9 +55,14 @@
     ## The view displayer generates HTML so we need to escape the XML special characters from the page reference.
     $escapetool.xml($pageReference)
   #else
-    #set ($title = $escapetool.xml($pageDoc.plainTitle))
+    #set ($title = $escapetool.velocity($pageDoc.plainTitle))
```

Comment:
This is wrong. You need XML escaping because you're printing the value below in HTML.